scripts/provisioning/s3setup/test_s3_setup.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
```

Comment:
I guess we should make this a python file, as we are trying to get away with shell scripts as much as possible.

Reply:
Considering this is a temporary script, can we let it be a shell script for the time being? Converting it to python is additional work and I feel we may not need to spend time on this conversion as the base script is already written, working and tested. And this script is also temporary.